_redirects

```diff
@@ -62,7 +62,7 @@
 /data/farming/grazing-improvement-area/                                                                   /data/farming/grazing-improvements/                                           301
 /data/farming/grazing-improvement-area/                                                                   /data/farming/grazing-improvements/                                           301
 /data/geoscience/aquifers-2/                                                                              /data/geoscience/aquifers/                                                    301
-/data/gis-data-sharing/                                                                                   /gis-data-sharing/                                                            301
+/data/gis-data-sharing/                                                                                   /sharing/                                                            301
```

Comment:
**nitpick**: we'll need a new entry for `/gis-data-sharing/` that goes to `/sharing/`